tests/test_import.py

```diff
@@ -28,4 +29,5 @@ def test_import_async_api():
         AsyncSessionDependency,
         SqlaAsyncSession,
         open_async_session,
+        startup,
```

Comment:
I think this should go to the test above since it's required for both sync and async sessions :)

Reply:
yeah, I was not sure where to put it